File: src/main/java/net/hectus/neobb/turn/card_game/CTOakDoor.java
package net.hectus.neobb.turn.card_game;

import net.hectus.neobb.player.NeoPlayer;
import org.bukkit.block.Block;

public class CTOakDoor extends InteractableCardTurn {
    public CTOakDoor(NeoPlayer player) { super(player); }
    public CTOakDoor(Block data, NeoPlayer player) { super(data, player); }

    @Override public double damage() { return 4; }
}

File: src/main/kotlin/net/hectus/neobb/modes/turn/card_game/CTOakDoor.kt
package net.hectus.neobb.modes.turn.card_game

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.player.NeoPlayer
import org.bukkit.block.Block

class CTOakDoor(data: Block?, cord: Cord?, player: NeoPlayer?) : InteractableCardTurn(data, cord, player) {
    override val damage: Double = 4.0
}
